packages/jats-to-myst/src/index.ts

```diff
@@ -0,0 +1,536 @@
+import type { Root } from 'myst-spec';
+import { unified } from 'unified';
+import doi from 'doi-utils';
+import type { Plugin } from 'unified';
+import { VFile } from 'vfile';
+import type { MessageInfo, GenericNode, GenericParent } from 'myst-common';
+import { toText, copyNode, fileError } from 'myst-common';
+import { select, selectAll } from 'unist-util-select';
+import { u } from 'unist-builder';
+import type { Handler, IJatsParser, JatsResult, Options, StateData } from './types';
+import { Jats, RefType } from 'jats-xml';
+import { keysTransform } from 'myst-transforms';
+import { basicTransformations } from './transforms';
+
+function refTypeToReferenceKind(kind?: RefType): string | undefined {
+  switch (kind) {
+    case RefType.sec:
+      return 'heading';
+    case RefType.fig:
+      return 'figure';
+    case RefType.dispFormula:
+      return 'equation';
+    case RefType.table:
+      return 'table';
+    case RefType.custom:
+      return undefined;
+  }
+}
+
+type Attributes = Record<string, any>;
+
+const handlers: Record<string, Handler> = {
```

Comment:
Lots of things still to turn on here, but the following is going quite well on much of the JATS I have found.